docs/internals/optimizer.rst

```diff
@@ -330,7 +330,7 @@ the ``--yul-optimizations`` option:
 The sequence inside ``[...]`` will be applied multiple times in a loop until the Yul code
 remains unchanged or until the maximum number of rounds (currently 12) has been reached.
 
-Available abbreviations are listed in the `Yul optimizer docs <yul.rst#optimization-step-sequence>`_.
+Available abbreviations are listed in the `Yul optimizer docs <https://docs.soliditylang.org/en/v0.8.13/yul.html#optimization-step-sequence>`_.
```

Comment:
This must be a reference, not a hard-coded link. Please use a proper reference. See [Cross-referencing with Sphinx > Explicit targets](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets)

Reply:
According to the doc you shared, the previous link looks correct to me. And I don't know properly how to reference other than the original one.

Reply:
Why does this link has to refer `yul.rst` in the first place? Should just refer to a section.

Reply:
> Why does this link has to refer `yul.rst` in the first place? Should just refer to a section.

If you know the correct reference, please paste here so I can update😄

Reply:
Link updated.